conda/shell/etc/fish/conf.d/conda.fish

```diff
@@ -93,7 +93,7 @@ function __fish_conda_commands
 end
 
 function __fish_conda_envs
-  eval $_CONDA_EXE env list | tail -n +3 | awk '{print $1}'
+  conda config --show envs_dirs | awk 'NR>1{print }' | xargs -IX find X -maxdepth 1 -mindepth 1 -type d -printf '%f\n' | sort
```

Comment:
It should be `awk 'NR > 1 {print $2}'`!

Reply:
Does the whitespace matter? It doesn't seem to on my system.

Reply:
It's not the whitespace, but the `$2` part.

Reply:
The whitespace is only for consistency with the command below (`__fish_conda_packages`) :)

Reply:
oh nevermind, I missed the `$2` somehow!